machineSetup/setupPerfMachine.ps1

```diff
@@ -0,0 +1,177 @@
+﻿#Set power scheme to high performance
+powercfg /S 8c5e7fda-e8bf-4a96-9a85-a6e23a8c635c
+powercfg /SETACVALUEINDEX SCHEME_MIN SUB_SLEEP STANDBYIDLE 0
+powercfg /SETACVALUEINDEX SCHEME_MIN SUB_VIDEO VIDEOIDLE 0
+powercfg /SETACVALUEINDEX SCHEME_MIN SUB_PROCESSOR PERFBOOSTMODE 0
+powercfg /SETACVALUEINDEX SCHEME_MIN SUB_PROCESSOR PROCTHROTTLEMAX 100
+powercfg /SETACVALUEINDEX SCHEME_MIN SUB_PROCESSOR PROCTHROTTLEMIN 100
+powercfg /SETACVALUEINDEX SCHEME_MIN SUB_PROCESSOR SYSCOOLPOL 1  
+powercfg /SETACVALUEINDEX SCHEME_MIN SUB_PROCESSOR THROTTLING 0  
+#region GIT Install Config
+$gitInstallLog = @"
+[Setup]
+Lang=default
+Dir=C:\Program Files\Git
+Group=Git
+NoIcons=0
+SetupType=default
+Components=ext,ext\shellhere,ext\guihere,assoc,assoc_sh
+Tasks=
+PathOption=Cmd
+SSHOption=OpenSSH
+CRLFOption=CRLFAlways
+BashTerminalOption=ConHost
+PerformanceTweaksFSCache=Enabled
+UseCredentialManager=Enabled
+EnableSymlinks=Disabled
+"@
+#endregion
+$gitInstallLog > gitInstallInfo.log
+wget https://github.com/git-for-windows/git/releases/download/v2.14.1.windows.1/Git-2.14.1-64-bit.exe -OutFile git-2.14.1.exe
+$gitArgList = "/LOADINF=gitInstallInfo.log", "/VERYSILENT"
+echo Start-Process -FilePath .\git-2.14.1.exe -ArgumentList $gitArgList -Wait
```

Comment:
We should replace the `echo` commands with `Write-Host`. The first command takes an array of strings and prints the on a different line, the second command will print a single line.